docs/ota.md

```diff
@@ -7,27 +7,873 @@ The messages are encoded as json payloads, and you can validate any payload by u
 Ruters repositories.  
 
 MQTT is used as the transport mechanism for all OTA messages. MQTT is a simple pub-sub based messaging protocol, that works on top of TCP/IP as well as on websockets. 
+Ruter has agreed with the operators to use only JSON in MQTT messages and not support Protobuf. The contents of the messages will therefore be defined by JSON schemas and these will be made available to the operators.
 
+?> This page is an update of an outdated OTA Messages wiki page that was based on version 1.0 of the requirements. Version 1.1 of the requirements introduced some changes and after the development process began, new needs were identified as documented here, partly for use in formulating a change order for the operators selected for routes in Romerike and the electric bus contract.
+
+## Summary
+The summary refers to topics on board the buses. See below for mapping of the local topic to the bridged topic.
+Changes noted in status are based on a comparison with v1.1 of the OTA Messages document. Direction is in/out of the bus.
+
+| Local topic                        | Change     | Direction   | Name                             | Comments                                                                                                                                                                                             |
+|------------------------------------|------------|-------------|----------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| signon/json                        |            | Out         | Start block                      |                                                                                                                                                                                                      |
```

Comment:
Is it possible/worth it to have the "Name" section as URIs that take you directly to the section in question? I remember you mentioned it was subdivided into messages, which I assume "Name" refers to?

Reply:
done